app/__init__.py

```diff
@@ -32,6 +32,9 @@ def create_app(config_name):
     from .main import main as main_blueprint
 
     application.register_blueprint(main_blueprint, url_prefix='/user')
+
+    # Must be registered last so that any routes declared in the app are registered first (i.e. take precedence over
+    # the external NotImplemented routes in the dm-utils external blueprint).
```

Comment:
#random

Reply:
I added it to all apps in the same place after registering external blueprints early caused things to break. See https://github.com/alphagov/digitalmarketplace-briefs-frontend/pull/77